Update to Android U

AjanZhong opened the pull request.

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

Commit:
Update to support Android U dessert

Removed plenty of patches which have been merged into AOSP Trusty
code base, and rebase remain patches based on Google Trusty code
base at Nov. 13th 2023.

Signed-off-by: Zhong,Fangjian <fangjian.zhong@intel.com>
Tracked-On: OAM-114118

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details

sysopenci commented:
Improper Commit Message
Improper Affects version

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/5301

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/5302

Done by "Zhong, Fangjian" <fangjian.zhong@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/3091

Done by "Zhong, Fangjian" <fangjian.zhong@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Zhong, Fangjian" <fangjian.zhong@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/3441
